controller/cache/info.go

```diff
@@ -190,6 +190,29 @@ func populatePodInfo(un *unstructured.Unstructured, node *node) {
 	if reason != "" {
 		node.info = append(node.info, v1alpha1.InfoItem{Name: "Status Reason", Value: reason})
 	}
+	node.health = createHealth(pod.Status, reason)
 	node.info = append(node.info, v1alpha1.InfoItem{Name: "Containers", Value: fmt.Sprintf("%d/%d", readyContainers, totalContainers)})
 	node.networkingInfo = &v1alpha1.ResourceNetworkingInfo{Labels: un.GetLabels()}
 }
+
+func createHealth(status v1.PodStatus, reason string) *v1alpha1.HealthStatus {
```

Comment:
Can we instead use `health.go::getPodHealth()` exporting it if necessary? If the logic there is not sufficient (e.g. it does not take into consideration `CrashLoopBackOff`), then we should improve that method.

Reply:
I will investigate

Reply:
ok, so both `getPodHealth` and `populatePodInfo` do the same thing and need to be merged